draft-birkholz-rats-corim.md

```diff
@@ -99,6 +99,52 @@ See {{-coswid}}.
 
 Privacy Considerations
 
+# Concise SWID Data Definition
+
+# 2.7 The link-entry Map
```

Comment:
Section numbering is created automatically.  At the MD level we only specify the heading nesting level by increasing the number of `#` signs prepended:

```suggestion
## The link-entry Map
```